crates/server-primitives/src/admin.rs

```diff
@@ -8,6 +9,13 @@ pub struct InstallApplicationRequest {
     pub version: semver::Version,
 }
 
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct InstallDevApplicationRequest {
+    pub application: calimero_primitives::application::ApplicationId, // TODO: rename to application_id
```

Comment:
I think you have the liberty to rename it now, since it's just now being introduced